### PR TITLE
Update Spanish.properties

### PR DESCRIPTION
**Pull request by Hello200000**
the only thing i didn't translate were the names of the cities if you have any problem contact me on discord like: lostxxxxx

### CODE REVIEW

**Comment by yairm210** [comment]
Placeholder `[tileFilter2]` not found in `Spanish` for entry `with [amount] to [amount2] neighboring [tileFilter] [tileFilter2] tiles`

**Comment by Hello200000** [comment]
WHERE THIS ERROR

File: android/assets/jsons/translations/Spanish.properties [hunk 1]
```diff
@@ -5530,174 +5529,175 @@ Often this results in the city immediately converting to their religion = A menu
...
+when above [amount] HP = cuando está por encima de [amount] HP
  # Requires translation!
-when below [amount] HP = 
+when below [amount] HP = cuando está por debajo de [amount] HP
  # Requires translation!
-with [amount] to [amount2] neighboring [tileFilter] tiles = 
+with [amount] to [amount2] neighboring [tileFilter] tiles = con [amount] a [amount2] [tileFilter] fichas
  # Requires translation!
-with [amount] to [amount2] neighboring [tileFilter] [tileFilter2] tiles = 
+with [amount] to [amount2] neighboring [tileFilter] [tileFilter2] tiles = con [amount] a [amount2] [tileFilter] [tileFilter] mosaicos
```
Comment (yairm210) [inline android/assets/jsons/translations/Spanish.properties:5658]: Here. The second tileFilter needs a '2' on the end :)

Comment (Hello200000) [reply android/assets/jsons/translations/Spanish.properties:5658]: thanks

Comment (Hello200000) [reply android/assets/jsons/translations/Spanish.properties:5658]: how can i edit it again?

Comment (Hello200000) [reply android/assets/jsons/translations/Spanish.properties]: I already learned to edit

Comment (Hello200000) [reply android/assets/jsons/translations/Spanish.properties]: I already resolved it

Comment (Hello200000) [reply android/assets/jsons/translations/Spanish.properties]: thanks for checking the document

**Pull request merged**